test/outtest9.F90

```diff
@@ -101,7 +103,11 @@ program outtest9
     if ( ier .ne. 2 ) stop 9
 
     ! Write the subset into the output message.
-    call writsb (21)
+    if ( ii.lt.3 ) then
+      call writsb (21)
+    else
+      call writsa (21, mxbfmg, mgbf, lmgbf)
```

Comment:
Are we checking this result somehow? Does it alter the output in any way?

Reply:
On that last pass of the loop, I'm using `writsa` as a replacement for `writsb`, which shouldn't and doesn't change the results. So the checking is the existing logic down below as far as number of messages and all that.

Reply:
Good work Alex - I like how you figured out how to test this using an existing test code!